packages/newspringchurchapp/ios/newspringchurchappTests/Info.plist

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+`<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what about this???

Reply:
wooooops

Reply:
I did this when merging into your `develop` branch I think